### PR TITLE
Added issue templates

### LINKED ISSUE #103: Create issue templates

### PR DESCRIPTION
**Pull request by eneoli**
Closes #103

### CODE REVIEW

**Comment by TornaxO7** [comment]
hm... could you maybe try to create issue templates by creating [issue-forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms)?

I could imagine to have a drop-down-menu where you can select the backend (X11 or wayland).

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        - Wayland
+        - X11
+      default: 0
+    validations:
+      required: true
+  - type: input
+    id: distro
+    attributes:
+      label: Which Linux Distro do you use?
+      placeholder: I use Arch btw.
```
Comment (TornaxO7) [inline .github/ISSUE_TEMPLATE/bug_report.yml:45]: perfect

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  - type: textarea
+    id: monitors
+    attributes:
+      label: How many monitors do you have and how are they orientated?
+      placeholder: I have two monitors both orientateed side to side in landscape.
```
Comment (TornaxO7) [inline .github/ISSUE_TEMPLATE/bug_report.yml:50]:
```suggestion
      placeholder: I have two monitors both orientateed side to side in landscape. (pls include images of the layout if possible)
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea to improve flakeshot!
+title: ''
+labels: 'feature request'
+assignees: ''
+
+---
```
Comment (TornaxO7) [inline .github/ISSUE_TEMPLATE/feature_request.md:8]: is this needed? What is this feature called in markdown? I haven't seen this one before.

**Review by TornaxO7 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 3 above)
Comment (eneoli) [reply .github/ISSUE_TEMPLATE/feature_request.md]: outdated.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 2 above)
Comment (eneoli) [reply .github/ISSUE_TEMPLATE/bug_report.yml:50]: Added to description.

**Review by TornaxO7 — APPROVED** [review]

**Pull request merged**